homeassistant/components/tuya/switch.py

```diff
@@ -408,6 +408,20 @@
             translation_key="switch",
         ),
     ),
+    # Irrigator (dual channel)
+    # https://developer.tuya.com/en/docs/iot/categoryggq?id=Kaiuz1qib7z0k
+    "ggq": (
+        SwitchEntityDescription(
+            key=DPCode.SWITCH_1,
+            translation_key="switch_zone_1",
+            entity_category=EntityCategory.CONFIG,
+        ),
+        SwitchEntityDescription(
+            key=DPCode.SWITCH_2,
+            translation_key="switch_zone_2",
+            entity_category=EntityCategory.CONFIG,
+        ),
```

Comment:
The documentation linked, does not have the DP codes that are used here; indicating this is a non-standard device.

We don't support non-standard devices. At minimal, we should support the standard (documented) instruction set.

Reply:
So based on this comment, you are rejecting this PR? Or do you think there's something which can be done to make this acceptable?

Since you do not want to support this device, how are people meant to use it?

Reply:
Is it possible to have this device working in Home Assistance Tuya integration?
It's disappointing that I cannot integrate it in my automations.

Reply:
> So based on this comment, you are rejecting this PR?

I didn't say that, nor did I close the PR. I said:

> At minimal, we should support the standard (documented) instruction set.